libraries/consul_service_windows.rb

```diff
@@ -64,12 +64,20 @@ def action_enable
       end
 
       def action_reload
-        Chef::Log.info 'The service provider for Consul on Windows does not support reload!'
+        notifying_block do
+          execute 'Reload consul' do
+            command 'consul.exe reload'
```

Comment:
Looks good, but I'm assuming you've tested this. Does it interfere with NSSM at all? Since consul is a child process of NSSM, I'd worry that NSSM would see the "`HUP`" & try launching it again (effectively being a restart).